spec/services/order_row_importer_spec.rb

```diff
@@ -641,6 +641,7 @@
           "Order",
           "Reference ID",
           "Errors",
+          "Project ID",
```

Comment:
Not sure how this works, but is this okay even if a core isn't using the Project extension?

Reply:
@LeticiaErrandonea this spec is in the engine too, so I just removed it here.

I explored updating it here to test the other columns, but since the engine is loaded, Project ID is always present